misk-aws/src/main/kotlin/misk/dynamodb/RealDynamoDbModule.kt

```diff
@@ -14,7 +13,8 @@ import javax.inject.Singleton
  * Install this module to have access to an AmazonDynamoDB client. This can be
  * used to create a DynamoDbMapper for querying of a DynamoDb table.
  */
-class RealDynamoDbModule : KAbstractModule() {
+class RealDynamoDbModule constructor(private val clientConfig: ClientConfiguration? = null) :
```

Comment:
Maybe make this non null and default to `ClientConfiguration()`? No need for the null check below

Reply:
Good call, I see that its default constructor invocation just sets every option to a default.. works for me!